packages/docs-reanimated/docs/debugging/logger-configuration.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: logger-configuration
+sidebar_label: Logger configuration
+sidebar_position: 2
+---
+
+# Logger configuration
+
+Reanimated shows warnings that indicate potential issues in the user code. These logs can be configured to be more or less verbose. The default configuration is to show all warnings. You can change this behavior by using the `configureReanimatedLogger` function.
+
```

Comment:
Let's give an example of such an warning here. Let's also make it more prominent that we warn of misuses of Reanimated API, than general "potential issues in the user code".

Reply:
You think, that including the example within a sentence (like: "Reanimated shows warnings that indicate misuses of the library API, such as modifying the Shared Value during component re-render") will be enough? I don't want to paste a screenshot with a warning in here as the description will become a bit messy.

Alternatively, I can add the section with example warning/warnings showing how they look like.

Reply:
Yes, I think such example would be sufficient 😄